### PR TITLE
Moved SPLASH_LIST to a seperate file

### PR DESCRIPTION
**Pull request by troy0h**
This should help clean up the MainForm.UI file a bit

### CODE REVIEW

File: AM2RLauncher/AM2RLauncher/MainForm/MainForm.UI.cs [hunk 1]
```diff
@@ -201,7 +163,7 @@ public MainForm()
             // Custom splash texts
             Random rng = new Random();
             // Some fanciness so we can have Gtk-only quips, because Gtk visuals are lame.
-            string splash = SPLASH_LIST[rng.Next(0, SPLASH_LIST.Length - (Platform.IsGtk ? 0 : 13))];
+            string splash = SPLASH.SPLASH_LIST[rng.Next(0, SPLASH.SPLASH_LIST.Length - (Platform.IsGtk ? 0 : 13))];
```
Comment (Miepee) [inline AM2RLauncher/AM2RLauncher/MainForm/MainForm.UI.cs:166]:
```suggestion
            string splash = Splash.SPLASH_LIST[rng.Next(0, Splash.SPLASH_LIST.Length - (Platform.IsGtk ? 0 : 13))];
```

File: AM2RLauncher/AM2RLauncher/MainForm/SPLASH_LIST.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AM2RLauncher
+{
+    class SPLASH
```
Comment (Miepee) [inline AM2RLauncher/AM2RLauncher/MainForm/SPLASH_LIST.cs:7]:
```suggestion
    class Splash
```

**Review by Miepee — COMMENTED** [review]
Feel like lowercase splash looks a bit cleaner

**Comment by Miepee** [comment]
Looks good to me.

**Comment by Lojemiru** [comment]
Hey! I really like the idea here, but the implementation doesn't go as far as it should towards abstraction. The splash list's calculation for Gtk-specific splashes should be offloaded to that class instead of being handled while getting the list. Ideally it'd just be a Splash.GetSplash() method that returns one chosen based on the current platform.

**Comment by troy0h** [comment]
This won't build at the moment, due to an issue with Platform.IsGtk that I can't seem to figure out

**Comment by troy0h** [comment]
GetSplash method is now working, and I also split the array, as Miepee asked, so it should be easier to add new splash text in the future

**Comment by Miepee** [comment]
Added some documentation and made the Splash class static, so it's not necessary anymore now to create an instance of it.

**Comment by Lojemiru** [comment]
This all checks out, sorry for the delay in getting to merging this.

**Pull request merged**